src/collections/Form/Form.js

```diff
@@ -7,8 +8,22 @@ export default class Form extends Component {
   static propTypes = {
     children: PropTypes.node,
     className: PropTypes.string,
+    settings: PropTypes.object,
   };
 
+  componentDidMount() {
+    this.element = $(this.refs.element);
+    this.element.form(this.props.settings);
+  }
+
+  componentWillUnmount() {
+    this.element.off();
+  }
+
+  plugin() {
+    return this.element.form(...arguments);
+  }
+
```

Comment:
Thanks for noting, we have not begun any performance optimizations.  I prefer to get all stable, then optimize.  Any help on this welcome.